meter/tasmota/connection.go

```diff
@@ -15,20 +15,31 @@ import (
 type Connection struct {
 	*request.Helper
 	uri, user, password string
+	Channel             int
 }
 
 // NewConnection creates Tasmota charger
-func NewConnection(uri, user, password string) (*Connection, error) {
+func NewConnection(uri, user, password string, channel int) (*Connection, error) {
 	if uri == "" {
 		return nil, errors.New("missing uri")
 	}
 
+	// Default Tasmota relay channel 1
+	if channel == 0 {
```

Comment:
Das kannst Du als Default oben in der Config machen

Reply:
Ok, hab ich rausgenommen. Default wird im configure template gesetzt.

Reply:
Der Default muss oben in das cc rein- sonst geht es nicht falls mal nicht das Template benutzt wird.